.github/workflows/foundry-tests.yml

```
@@ -122,6 +110,17 @@ jobs:
         with:
           version: nightly
 
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)